IBPSA/Resources/Data/schedule.txt

```diff
@@ -0,0 +1,6 @@
+#234asdfasdfasdfasdfasdfljksdfklajslkdfjaskdfjkasdk;lfajksldfjasdkfjas;dfkjasklf;asdkfaksdjf;asdjkfajskd
```

Comment:
Is there a better way to explain that this is a comment rather than 234asfklafj;aabj;ab?

Reply:
😅 I'm not sure but I can give it a try =)